docs/en/ingest-management/elastic-agent/otel-agent.asciidoc

```diff
@@ -0,0 +1,145 @@
+* Run a processor to insert `elastic-otel-test` into the output.
+* Send the data with a `detailed` level of verbosity.
++
+Refer to <<otel-agent-configuration>> to learn more about setting up this file.
+
+. [Optional] Override a setting in the OTel configuration
++
+You can use the `set` flag to manually override any setting defined in the OTel configuration file. For example, run the following command to change the default verbosity setting from `detailed` to `normal`.
++
+[source,shell]
+----
+sudo ./elastic-agent otel --config otel.yml --set "exporters::debug::verbosity=normal"
+----
+
+. Validate the configuration
++
+To validate the OTel configuration for your {agent}, run the `otel` command with the `validate` option and the path to the config file:
++
+[source,shell]
+--
+sudo ./elastic-agent otel validate --config otel.yml
+--
++
+If no error is returned, the configuration file is determined to be valid.
+
+. Run {agent}
++
+Now, run {agent} with OTel mode configured:
++
+[source,shell]
+--
+sudo ./elastic-agent otel --config otel.yml
+--
++
+You should see a response indicating that {agent} has started successfully in OTel mode:
++
+[source,shell]
+--
+Starting in otel mode
+2024-05-31T12:39:43.047-0400	info	service@v0.94.1/telemetry.go:59	Setting up own telemetry...
+2024-05-31T12:39:43.049-0400	info	service@v0.94.1/telemetry.go:104	Serving metrics	{"address": ":8888", "level": "Basic"}
+2024-05-31T12:39:43.052-0400	info	exporter@v0.94.1/exporter.go:275	Development component. May change in the future.	{"kind": "exporter", "data_type": "logs", "name": "debug"}
+2024-05-31T12:39:43.055-0400	info	service@v0.94.1/service.go:140	Starting ./elastic-agent...	{"Version": "8.13.4", "NumCPU": 10}
+2024-05-31T12:39:43.055-0400	info	extensions/extensions.go:34	Starting extensions...
+2024-05-31T12:39:43.056-0400	info	adapter/receiver.go:45	Starting stanza receiver	{"kind": "receiver", "name": "filelog", "data_type": "logs"}
+2024-05-31T12:39:43.057-0400	info	service@v0.94.1/service.go:166	Everything is ready. Begin running and processing data.
+2024-05-31T12:39:43.057-0400	warn	localhostgate/featuregate.go:63	The default endpoints for all servers in components will change to use localhost instead of 0.0.0.0 in a future version. Use the feature gate to preview the new default.	{"feature gate ID": "component.UseLocalHostAsDefaultHost"}
+2024-05-31T12:39:43.261-0400	info	fileconsumer/file.go:261	Started watching file	{"kind": "receiver", "name": "filelog", "data_type": "logs", "component": "fileconsumer", "path": "/var/log/system.log"}
+--
+
+. Check the {agent} status
++
+Finally, confirm that {agent} is running as expected. Open a second terminal shell and run the `status` command.
++
+[source,shell]
+--
+sudo ./elastic-agent status
+--
++
+The returned status indicates that {agent} is healthy and not enrolled in {fleet} (which is expected because in this example we're using standalone {agent}).
++
+[source,shell]
+--
+┌─ fleet
```

Comment:
@ycombinator is elastic-agent status going to report any issues with the otel mode collector?

Reply:
No, it will not report any issues with the embedded OTel collector.  Here's what the output of `./elastic-agent status` looks like when Agent is running in OTel mode (in a separate tab/window):

```
$ ./elastic-agent status --output full
┌─ fleet
│  └─ status: (STOPPED) Not enrolled into Fleet
└─ elastic-agent
   ├─ status: (HEALTHY) Running
   └─ info
      ├─ id:
      ├─ version: 8.14.0
      └─ commit: d0bce03b68012224ac1bab4ca20ba36060d135a3
```

Reply:
I've updated the docs with the `./elastic-agent status --output full` command (as opposed to just `./elastic-agent status`).

---

![Screenshot 2024-06-18 at 12 13 50 PM](https://github.com/elastic/ingest-docs/assets/41695641/88dd7ade-7776-4336-b28c-e625008b5c9c)